### PR TITLE
Dev -> Master for v1.13.2 release

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -4,47 +4,48 @@ name: Python tests
...
-    - uses: codecov/codecov-action@v1
-      name: Upload code coverage report
-      with:
-        if: success()
-        token: ${{ secrets.CODECOV_TOKEN }}
+        run: |
+          mkdir /tmp/nextflow
+          cd /tmp/nextflow
+          wget -qO- get.nextflow.io | bash
+          sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
```
Comment (ewels) [inline .github/workflows/pytest.yml:42]:
```suggestion
          wget -qO- get.nextflow.io | bash
          sudo mv nextflow /usr/local/bin/
```

**Review by KevinMenden — APPROVED** [review]
Alright, been through it, looks good!
Fixes quite a lot of bugs 👍

**Review by drpatelh — APPROVED** [review]
DOOOOO ITTT!!!

**Review by ggabernet — APPROVED** [review]
Looks good to me!

**Pull request merged**